experimental/packages/sdk-logs/src/types.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { IResource } from '@opentelemetry/resources';
+
+export interface LoggerConfig {
```

Comment:
> I think it would be better to call this `LoggerProviderConfig` since it is passed into the provider. `forceFlushTimeoutMillis` is also a provider-only configuration.

I've changed here,  split LoggerProviderConfig and LoggerConfig:  https://github.com/open-telemetry/opentelemetry-js/pull/3549/commits/e8ced536a7c7d47b7b7ee1b0e24e3ffabc64e6b4